app/views/home/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Ahoy, me hearties! Be ye welcome to Treasure Trove.</h2>
+
```

Comment:
Something Demarcus and I removed from our first draft of the home page was a conditional message encouraging the user to login or register if not logged in. We can return to that later after we resolve the various sad paths.